libs/js-sdk/src/utils/openTelemetryTracePassthrough.ts

```diff
@@ -0,0 +1,6 @@
+import { http, passthrough } from 'msw'
+
+export const openTelemetryTracePassthrough = http.post(
+    'http://localhost:4318/v1/traces',
```

Comment:
Is this meant to be hardcoded?

Reply:
Yeah, it's just for unit tests if you configure open telemetry locally using Jaeger it will pass the trace calls through.

This wouldn't work if you wanted to use honeycomb or app insights for unit test telemetry though